modules/lib/src/main/resources/assets/js/app/view/context/widget/details/ScheduleWizardStepForm.ts

```diff
@@ -66,6 +84,20 @@ export class ScheduleWizardStepForm
         });
     }
 
+    private findFirstChildWithClassIn(parent: Element, className: string): Element {
+        for (const child of parent.getChildren()) {
+            if (child.hasClass(className)) {
+                return child;
+            } else {
+                let found = this.findFirstChildWithClassIn(child, className);
+                if (found) {
+                    return found;
+                }
+            }
+        }
+        return null;
```

Comment:
This can be replaced with 
```Typescript
return parent.getChildren().find((child) =>
    child.hasClass(className) || this.findFirstChildWithClass(child, className));
```

Reply:
No, probably not exactly. The second part should be the element returned by `findFirstChildWithClass`

Reply:
Won't work: will return top most parent containing required element, not element itself

Reply:
Ok, but then let's at least simplify it as this is a classic recursion:
```Typescript
private findFirstChildWithClass(parent: Element, className: string): Element {
    for (const child of parent.getChildren()) {
        if (child.hasClass(className)) {
            return child;
        }

        return this.findFirstChildWithClass(child, className);
    }
    
    return null;
}
```

Reply:
Won't work also - will look result always only in first child